contracts/PushComm/PushCommV3.sol

```diff
@@ -615,24 +615,20 @@ contract PushCommV3 is Initializable, PushCommStorageV2, IPushCommV3, PausableUp
         whenNotPaused
     {
         // Implement restrictions based on functionType
-        if (functionType == CrossChainRequestTypes.CrossChainFunction.AddChannel) {
+
+        if (uint8(functionType) < 3) {
```

Comment:
Even though this is more gas-optimised, but is not better for future maintainability